src/modules/measurements/MeasurementItem.js

```diff
@@ -59,31 +72,35 @@ const MeasurementItem = (props) => {
     return measurementText === '' ? '?' : measurementText;
   };
 
+  const onMeasurementPress = () => {
+    if (isMultipleFeaturesTaggingEnabled) setFeatureSelectedForTagging(useTags.setFeaturesSelectedForMultiTagging(props.item));
+    else props.onPress();
+  };
+
   return (
     <React.Fragment>
       {typeof (props.item) !== 'undefined' && (
         <ListItem
           containerStyle={props.selectedIds.includes(props.item.id) ? commonStyles.listItemInverse
-            : commonStyles.listItem}
+            : [commonStyles.listItem, {
+              backgroundColor: featureSelectedForTagging
+                ? themes.PRIMARY_ACCENT_COLOR : themes.SECONDARY_BACKGROUND_COLOR,
+            }]}
           key={props.item.id}
-          onPress={() => props.onPress()}
+          onPress={() => onMeasurementPress()}
           pad={5}
         >
           <ListItem.Content>
             <ListItem.Title
               style={props.selectedIds.includes(props.item.id) ? commonStyles.listItemTitleInverse
                 : commonStyles.listItemTitle}
-            >
-              {getMeasurementText(props.item)}
-            </ListItem.Title>
-          </ListItem.Content>
-          <ListItem.Content>
-            <ListItem.Title
-              style={props.selectedIds.includes(props.item.id) ? commonStyles.listItemTitleInverse
-                : commonStyles.listItemTitle}
-            >
-              {getTypeText(props.item)}
+             >
+              <View style={{flexDirection: 'row', alignItems:'center'}}>
```

Comment:
This looks more complicated than I think it needs to be. How about just 

          <ListItem.Content>
            <ListItem.Title
              style={props.selectedIds.includes(props.item.id) ? commonStyles.listItemTitleInverse
                : commonStyles.listItemTitle}
            >
              <Text>{getMeasurementText(props.item)} {getTypeText(props.item)}</Text>
            </ListItem.Title>
            <FeatureTagsList spotId={spot.properties.id} featureId={props.item.id}/>
          </ListItem.Content>

Reply:
taken care.